### PR TITLE
Sprint 29

### PR DESCRIPTION
**Pull request by hopetambala**
### Description of proposed changes
What does this pull request accomplish? Why should it be integrated?

### Checklist
_Put an `x` in the boxes that apply. _
- [ ] My lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my changes work (if applicable)
- [ ] I have added or updated necessary documentation (if appropriate)

### Additional comments
Is there anything else you want me to know?

### CODE REVIEW

File: cloud/src/definer/crud.definer.js [hunk 1]
```diff
@@ -166,16 +166,16 @@ Parse.Cloud.define('postObjectsToClassWithRelation', (request) => new Promise((r
   let loopedJson = {};
   let newFieldsArray = [];
   Object.keys(localObject).forEach((key) => {
-    const obj = localObject[key];
-    if (!obj.includes('data:image/jpg;base64,')) {
```
Comment (hopetambala) [inline cloud/src/definer/crud.definer.js:170]: @jamccomb92 

not all they keys are `Strings` so if you tried to `includes()` on an object or an array, this broke.

**Review by hopetambala — COMMENTED** [review]
solid enough

File: cloud/src/_utils/offline/household_relations.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const HouseholdRelation = {
       if (householdsRelation !== null && householdsRelation !== undefined) {
         // post new households with relations to existing households
         householdsRelation.forEach((householdRelation, index, array) => {
-          if (!householdRelation.parseParentClassID.includes('Household-')) {
+          if (householdRelation.parseParentClassID.indexOf('Household-') < 0) {
```
Comment (hopetambala) [inline cloud/src/_utils/offline/household_relations.js:21]: since these are will always be strings, `indexOf` will work

**Review by jamccomb92 — COMMENTED** [review]
this all makes sense. understood.

**Pull request merged**